curator-recipes/src/main/java/org/apache/curator/framework/recipes/cache/TreeCache.java

```diff
@@ -704,8 +687,7 @@ private TreeNode find(String findPath)
             {
                 TreeNode childNode = entry.getValue();
                 ChildData childData = childNode.childData;
-                // Double-check liveness after retreiving data.
```

Comment:
I kind of liked this comment, although I misspelled "retrieving" here and below

Reply:
The reason I removed it is that unlike before it's not really "double" checking anymore against the separate `nodeState` field, there's only a single check for liveness which is now implicit in the value of `childData`. But I could well have misinterpreted the reason for comment, no problem with leaving it in!